Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:16.04
+RUN apt-get update && apt-get install -y git-core openssh-client curl wget \
+ build-essential openssl libreadline6 libreadline6-dev curl zlib1g zlib1g-dev \
+ libssl-dev libyaml-dev libsqlite3-dev sqlite3 libxml2-dev libxslt-dev \
+ autoconf libc6-dev ncurses-dev automake libtool bison pkg-config \
+ gawk libgmp-dev libgdbm-dev libffi-dev
+RUN curl -L https://get.rvm.io | bash -s stable
+RUN /bin/bash -l -c "rvm requirements"
```

Comment:
If you add this:

```sh
echo "gem: --no-document" >~/.gemrc
```

The gem/bundle commands will skip installing docs which will speed them up by 2-3x